### PR TITLE
Docs: add explanation of the base `.btn` class

### PR DESCRIPTION
**Pull request by patrickhlauke**
### Description

As discussed recently in the team slack, this adds an initial explanation of the intention behind the basic `.btn` class - as a starting point for either using the extra predefined styles, or for an author's own stylings.

Also adds a callout reminding authors to at least define some focus styling if they intend to use it "naked".

### Motivation & Context

We're often seeing authors using just `.btn`, and this "naked" use is not actually documented/explained anywhere. Also, authors then forgetting to create focus styles causes accessibility issues further down the line if they're not expecting it. We may decide in future to still add some absolutely basic focus styling, just in case, but this would be a breaking change for the next minor release/

#### Live previews

* https://deploy-preview-37275--twbs-bootstrap.netlify.app/docs/5.3/components/buttons/

### CODE REVIEW

File: site/content/docs/5.2/components/buttons.md [hunk 1]
```diff
@@ -6,7 +6,19 @@ group: components
 toc: true
 ---
 
-## Examples
```
Comment (julien-deramond) [inline site/content/docs/5.2/components/buttons.md:9]: IMO we should keep a header between this part (called "About" or "Overview" or something else) in order to be consistent with our other components; we never have a component whose doc begins without any h1 or h2.

**Review by julien-deramond — COMMENTED** [review]
This explanation looks good to me and explain well what are "naked" buttons and how they look like. It will help with non-regression testing.

@mdo I let you check it too

File: site/content/docs/5.2/components/buttons.md (hunk 1 above)
Comment (patrickhlauke) [reply site/content/docs/5.2/components/buttons.md:9]: ah, good point. i'll have a think what would work best there...

Comment (patrickhlauke) [reply site/content/docs/5.2/components/buttons.md:9]: went for "Base class" as an initial heading

**Review by julien-deramond — APPROVED** [review]

File: site/content/docs/5.2/components/buttons.md [hunk 2]
```diff
@@ -6,7 +6,21 @@ group: components
...
+Bootstrap has a base `.btn` class that sets up basic styles such as padding and content alignment. By default, `.btn` controls have a transparent border and background color, and lack any explicit focus and hover styles.
+
+{{< example >}}
+<button type="button" class="btn">Base class</button>
+{{< /example >}}
+
+The `.btn` class is intended to be used in conjunction with our additional predefined button styles, or to serve as a basis for your own custom styles.
+
+{{< callout warning >}}
+If you are using the `.btn` class on its own, remember to at very least define some explicit `:focus` and/or `:focus-visible` styles.
```
Comment (XhmikosR) [inline site/content/docs/5.2/components/buttons.md:20]: Not a native English speaker, I'm wondering if this should read "at the very least"?

Comment (patrickhlauke) [reply site/content/docs/5.2/components/buttons.md:20]: ah, good catch

**Comment by patrickhlauke** [comment]
@mdo when you get a moment, could you give this a thumbs up or down?

**Comment by patrickhlauke** [comment]
Going to be so bold as to merge it, as there doesn't seem to be any further feedback on it

**Pull request merged**